pkg/controllers/tagging/tagging_controller_test.go

```diff
@@ -70,6 +70,54 @@ func Test_NodesJoiningAndLeaving(t *testing.T) {
 			toBeTagged:       true,
 			expectedMessages: []string{"Successfully tagged i-0001"},
 		},
+		{
+			name: "node0 joins the cluster and was tagged earlier with different tags.",
+			currNode: &v1.Node{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "node0",
+					CreationTimestamp: metav1.Date(2012, 1, 1, 0, 0, 0, 0, time.UTC),
+					Labels: map[string]string{
```

Comment:
Can you have a case with >1 labels as well?

Reply:
Did you mean where multiple labels exist already on the node? Because I already modified the tests to have two labels and added a test where one of the labels was already present on the node.

Reply:
Yeah that covers it, thanks. :+1: